python-avd/pyavd/_eos_cli_config_gen/j2templates/eos-intended-config.j2

```diff
@@ -86,6 +86,8 @@
 {% include 'eos/match-list-input.j2' %}
 {# mcs client #}
 {% include 'eos/mcs-client.j2' %}
+{# monitor server radius #}
+{% include 'eos/monitor-server-radius.j2' %}
```

Comment:
can you confirm that monitor-server-radius config order with platform trident config

Reply:
Platform trident command are not configured on vEOS.

Reply:
Confirm on trident. This order is correct.